core_weekly/template.py

```diff
@@ -315,6 +315,9 @@ def build_merged_pull_requests(self, result):
             for category, items in sorted_category_items.items():
                 category_name = CATEGORIES[category] if category in CATEGORIES.keys() else category
 
+                if (category in category_blacklist):
```

Comment:
```suggestion
                if category in CATEGORY_BLACKLIST:
```
There is no parenthesis in python (if not needed)